addons/l10n_tr_nilvera_einvoice/models/account_move.py

```diff
@@ -142,48 +174,66 @@ def _l10n_tr_nilvera_get_submitted_document_status(self):
                                 "%s<br/>%s - %s<br/>"
                             ) % (
                                 _("The invoice couldn't be sent to the recipient."),
-                                response['InvoiceStatus'].get('Description'),
-                                response['InvoiceStatus'].get('DetailDescription'),
+                                response.get('InvoiceStatus', {}).get('Description') or response.get('StatusDetail'),
+                                response.get('InvoiceStatus', {}).get('DetailDescription') or response.get('ReportStatus'),
                             )
                         )
                 else:
                     invoice.message_post(body=_("The invoice status couldn't be retrieved from Nilvera."))
 
-    def _l10n_tr_nilvera_get_documents(self):
+    def _l10n_tr_nilvera_get_documents(self, invoice_channel, document_category, journal_type):
```

Comment:
Changing this in stable is problematic, it would break the customization on this, use a default value at least